src/map/atcommand.c

```diff
@@ -126,6 +126,7 @@ static const char *atcommand_msg(int msg_number)
 	if(atcommand->msg_table[0][msg_number] != NULL && atcommand->msg_table[0][msg_number][0] != '\0')
 		return atcommand->msg_table[0][msg_number];
 
+	ShowWarning("atcommand_msg: Invalid message number was specified: %d", msg_number);
```

Comment:
probably need also show stack, because can be too hard find source of wrong message id.

for this you can add ``Assert_report(0)``

Reply:
Done, thanks :)